packages/theming/theme-tokens/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export { default as globalTokens } from './tokens-global';
+export { default as aliasTokens } from './light-macos/tokens-alias';
```

Comment:
I would rather us expose alias tokens through an implementation of `getAliasTokens` per platform. That way we allow for the alias tokens to change depending on light/dark/HC mode, and there's only one point exposed to get the alias tokens. If we export alias tokens directly like this, anyone else who imports `aliasTokens` instead of using `getAliasTokens` would only get the macOS light version ones.

Reply:
@rurikoaraki Thank you for the suggestion, added `getAliasTokens` implementation for macOS in 80f44e276ce34b9db6a78deeae6350eeb5a4e24b. Please take a second look when you get a chance.

Reply:
Thanks @chiuam! Is there a reason to keep this implementation separate from the other `getAliasTokens`? Could we make a `getAliasTokens.macos.ts` so that `getAliasTokens` changes what it does on macos automatically instead of exporting an additional function?

Office has its own version as the function signature has to be different from the base `getAliasTokens`, but it looks like the Mac one can match the signature, unless you think this will change in the future.

Reply:
Oh yeah, I didn't think of that. Fixed in b0aa4517334d4780e2c24a1236028f4730394a74

Reply:
Removed aliasTokens export in 7492c9ef0313fe93d0017ba534cfbb827efa69e4

Reply:
You're still exporting it here though, I think we can remove this, too

Reply:
removed, for real this time :)

Reply:
^I removed the wrong one, should be fixed now...

Reply:
Thanks!